include/clang/Sema/Overload.h

```diff
@@ -193,6 +193,17 @@ namespace clang {
     /// \brief Whether this binds a reference to an object with a different
     /// Objective-C lifetime qualifier.
     unsigned ObjCLifetimeConversionBinding : 1;
+
+    /// \brief e.g. conversions from pointer -> capability without an explicit __cheri_cast
+    unsigned IncompatibleCHERIConversion : 1;  // XXXAR: would be nice if we had a ctor to initialize this
```

Comment:
Does `= 0` not do the right thing WRT constructors?  Also, is there a reason that this is not a `bool`?

Reply:
`= 0` doesn't work because this type can't have a non-trivial constructor due to the way it is used. Also I'm not sure it works for bitfields before C++17